source/AI.cpp

```diff
@@ -1085,8 +1113,13 @@ void AI::AskForHelp(Ship &ship, bool &isStranded, const Ship *flagship)
 			if((helper->IsYours() && !ship.IsYours()) || helper.get() == flagship)
 				continue;
 			// Your escorts should not help each other if already under orders.
-			if(helper->IsYours() && ship.IsYours() && orders.count(helper.get()))
-				continue;
+			auto foundOrders = orders.find(helper.get());
+			if(foundOrders != orders.end())
+			{
+				int helperOrders = foundOrders->second.type;
+				if(helper->IsYours() && ship.IsYours() && helperOrders != Orders::MINE && helperOrders != Orders::HARVEST)
+					continue;
```

Comment:
@tehhowch these two lines were added as part of earlier feedback from you that you don't want harvest to override other actions such as attacking enemies.

I think this should be removed now that Harvest is a full featured command similar to attack (F) and other orders.  i.e. if the player has issued orders to Harvest the fleet should focus on harvesting and not attacking.  This would bring it more in-line with other command orders the player issues.

Thoughts?

Reply:
Nevermind, this was an incorrect statement at the wrong piece of code.

The harvest order acts as a proper order and fleet ignores enemies when harvest order is active.  That was done when the shortcut for harvest orders was broken out into a separate command.